### DISCUSSION TITLE
Lack of documentation for version 7

### DISCUSSION THREAD

**Discussion by pthreat** [ruflin/Elastica#1968] [General]
Examples provided here:

https://elastica.io/getting-started/storing-and-indexing-documents.html

Do not work for version 7

**Comment by ruflin** [comment]
I must confess, the docs did not get the attention they deserve over the past few years. This page is pretty telling: https://github.com/ruflin/Elastica.io/tree/master/source I wonder if it would be better to wipe the docs as they are outdated and because of this contain false information.
